Documentation/contributing/development/dev_setup.rst

```diff
@@ -692,6 +692,54 @@ change:
     $ make generate-k8s-api
     $ git add go.mod go.sum vendor/
 
+Add/update a cilium/kindest-node image
+--------------------------------------
+
+Cilium might use its own fork of kindest-node so that it can use k8s versions
+that were not yet released by Kind maintainers.
+
+One other reason is the base image used on kindest-node, which has not been yet
+released: Debian Bookworm. Cilium requires Bookworm because the glibc version
+being available on Cilium's base docker image is the same as the one used in the
+Bookworm docker image which is relevant for testing with Golang's race detector.
+
+Currently, to publish an image on ``quay.io/cilium/kindest-node`` one has to be
+a maintainer. However, anyone can build a kindest-node image and try it out
+on a Pull Request.
+
+First, we need to build the base docker image and for that we need to clone the
+kind repository https://github.com/kubernetes-sigs/kind and run
+
+   .. code-block:: shell-session
+
+    make -C images/base/ quick
+
+Note the resulting image tag for that command, it should be last tag built for
+the ``gcr.io/k8s-staging-kind/base`` repository.
+
+Secondly, change into the directory with Kubernetes' source code which will be
+used for the kindest node image. On this example, we will build a kindest-base
+image with Kubernetes version ``v1.28.3`` using the recently build base image
+``gcr.io/k8s-staging-kind/base:v20231108-a9fbf702``:
+
+   .. code-block:: shell-session
+
+    $ # Change to k8s' source code directory.
+    $ # git clone https://github.com/kubernetes/kubernetes.git
+    $ # cd kubernetes
```

Comment:
Why the `#`? Users can just skip these if they've run it already, they're not comments?

```suggestion
    $ git clone https://github.com/kubernetes/kubernetes.git
    $ cd kubernetes
```

Reply:
fair point